stdlib/public/SDK/Foundation/NSStringAPI.swift

```diff
@@ -1023,20 +1027,19 @@ extension String {
     scheme tagScheme: String,
     options opts: NSLinguisticTaggerOptions = [],
     orthography: NSOrthography? = nil,
-    tokenRanges: UnsafeMutablePointer<[Range<Index>]> = nil // FIXME:Can this be nil?
+    tokenRanges: UnsafeMutablePointer<[Range<Index>]>? = nil // FIXME:Can this be nil?
   ) -> [String] {
     var nsTokenRanges: NSArray? = nil
-    let result = tokenRanges._withBridgeObject(&nsTokenRanges) {
+    let op = {
```

Comment:
@jrose-apple Making this instance easier to read is not good enough justification to repeat open-coded logic that is already encapsulated in an idiom we use throughout the library.